README.rst

```diff
@@ -1,5 +1,31 @@
-reclass README
-==============
+reclass (extended) README
+=========================
 
-The documentation for **reclass** is available from
-http://reclass.pantsfullofunix.net.
+This is an extended version of **reclass** available at:
+https://github.com/AndrewPickford/reclass
```

Comment:
I would only state That README.extensions.rst contains features added above the reclass.pantsfullofunix.net version.

We might tend to ship this for as package one day? As well as publish it's generated doc, so link should go here as well.